js/src/bitfinex2.js

```diff
@@ -1521,7 +1517,7 @@ export default class bitfinex2 extends Exchange {
         if ((ioc || fok) && exchangeMarket) {
             throw new InvalidOrder(this.id + ' createOrder() does not allow market IOC and FOK orders');
         }
-        if ((orderType !== 'MARKET') && (!exchangeMarket) && (!exchangeStop)) {
+        if ((orderType !== 'EXCHANGE MARKET') && (!exchangeMarket) && (!exchangeStop)) {
```

Comment:
Hello @lfern, this file (.js) is automatically generated from the typescript version (.ts), can you revert this one and apply the changes there?

Reply:
Sorry, @carlosmiei I didn't see your message. I will try to do it in .ts file.